operator/v1/types_clustermanager.go

```diff
@@ -135,6 +135,12 @@ type RegistrationDriverHub struct {
 	// For awsirsa authentication type, AutoApprovedIdentities represent a list of approved arn patterns
 	// +optional
 	AutoApprovedIdentities []string `json:"autoApprovedIdentities,omitempty"`
+
+	// List of tags to be added to AWS resources created by hub while processing awsirsa registration request.
+	// Applicable to only awsirsa authentication type.
+	// Example - "product:v1:tenant:app-name=My-App"
```

Comment:
I am just wondering is `app-name=My-App` close to the real usage of this field? If it is, please ignore this comment. Is better to have example like primary/secondary hub? Just guessing.

Reply:
Yes appname is the real usage, we have tags like appname, createdby, deptname etc

We are planning to reuse same roles for both primary and secondary, which is why we are not adding primary/secondary for now.

Reply:
I can't resolve this thread. Please feel free to resolve it if you are able to. Thanks.